Tests/SwiftDocCTests/Infrastructure/DocumentationContext/DocumentationContextTests.swift

```diff
@@ -644,15 +644,25 @@ class DocumentationContextTests: XCTestCase {
     
     func testDetectsReferenceCollision() throws {
         let (_, context) = try testBundleAndContext(named: "TestBundleWithDupe")
-        
-        XCTAssert(
-            context.problems.contains { problem in
-                problem.diagnostic.identifier == "org.swift.docc.DuplicateReference"
-                    && problem.diagnostic.localizedSummary == "Redeclaration of 'TestTutorial.tutorial'; this file will be skipped"
-            }
-        )
+
+        let problemWithDuplicate = context.problems.filter{ $0.diagnostic.identifier == "org.swift.docc.DuplicateReference" }
+
+        XCTAssertEqual(problemWithDuplicate.count, 1)
+
+        XCTAssertEqual(problemWithDuplicate[0].diagnostic.localizedSummary, "Redeclaration of 'TestTutorial.tutorial'; this file will be skipped")
```

Comment:
This line will crash if the previous assertion fails. You can use `try XCTUnwrap(problemWithDuplicate.first?.…` instead so that XCTest provides an assertion failure instead.

Reply:
make sense, i added that change.

Reply:
it was not pushed, i did this morning.